lightwood/encoders/categorical/__init__.py

```diff
@@ -1,6 +1,3 @@
 from lightwood.encoders.categorical.onehot import OneHotEncoder
+from lightwood.encoders.categorical.multihot import MultiHotEncoder
 from lightwood.encoders.categorical.autoencoder import CategoricalAutoEncoder
-
-default = CategoricalAutoEncoder
-onehot = OneHotEncoder
-autoencoder = CategoricalAutoEncoder
```

Comment:
Are these (except the `default` which is replaced by this PR) used anywhere, guys? Tell me if they are, then I will bring these back